frontend/packages/dev-console/src/components/pipelines/pipeline-builder/utils.ts

```diff
@@ -93,6 +95,13 @@ const removeEmptyDefaultParams = (task: PipelineTask): PipelineTask => {
   return task;
 };
 
+export const removeEmptyDefaultValueFromParameter = (parameters: PipelineParam[]) =>
```

Comment:
```suggestion
export const removeDefaultFromParam = (parameters: PipelineParam[]) =>
```

Reply:
we want to remove `default` only if it's empty hence the name `removeEmptyDefaultValueFromParameter`

Reply:
Yeah, but the name seems too big and can be shortened. If you can then it will be good otherwise we can let this be.

Reply:
changed it to `removeEmptyDefaultFromPipelineParams`